package.json

```diff
@@ -4,28 +4,26 @@
   "version": "1.0.0",
   "description": "Discord teams bot",
   "main": "src/index.ts",
-  "type": "module",
   "engines": {
     "node": ">=18.x"
   },
   "scripts": {
-    "build": "tsc && npm run postbuild",
-    "postbuild": "node replace-imports.js",
```

Comment:
does removing type module mean we don't need that postbuild script?

Reply:
works on my end